enterprise/admin/views.py

```diff
@@ -213,7 +219,7 @@ def _handle_bulk_upload(cls, enterprise_customer, manage_learners_form, request)
                     list_of_emails=", ".join(duplicate_emails)
                 )
             )
-        return list(emails) + already_linked_emails
+        return list(emails) + [email for email, __ in already_linked_emails]
```

Comment:
Nitpicky, but I prefer simple return statements with the work being done at an earlier point in the function, which makes testing + debugging easier, imho.